.scripts/supervisord.conf

```diff
@@ -25,4 +28,17 @@ stopsignal=INT
 autostart=true
 autorestart=true
 redirect_stderr=true
-priority=10
\ No newline at end of file
+priority=10
+
+[program:warp-input]
+command=/usr/bin/warp-input --namespace %(SESSION_ID_INPUT)s https://fst.so:4443
```

Comment:
What is the fst.so domain here?

That domain doesn't appear to have a valid website (at present anyway), so it's not easy to tell wtf this is. :smile:

Reply:
the link you mentioned directs to our public relays or CDN, which hosts the netris/relay. It's not designed to function as a conventional website or respond to standard REST requests. Its sole purpose is to intercept QUIC, webtransport, and datagrams, enabling clients to connect and subscribe to video/audio from the server.

Reply:
> Hey @AToska21 the link you mentioned directs to our public relays or CDN, which hosts the netris/relay. It's not designed to function as a conventional website or respond to standard REST requests. Its sole purpose is to intercept QUIC, webtransport, and datagrams, enabling clients to connect and subscribe to video/audio from the server.

AToska21 asked about self hosting :)

Reply:
Hey @ellidi 

😅 My bad!

Reply:
@wanjohiryan No worries at all.  I just tend to ask questions when I see PRs adding undocumented domains.  Basic precautionary "just in case" kind of thing. :smile: